rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/are_all_terminal_chw_loops_purcahsed_cooling.py

```diff
@@ -0,0 +1,51 @@
+from rct229.data.schema_enums import schema_enums
+from rct229.ruleset_functions.baseline_systems.baseline_system_util import (
+    find_exactly_one_fluid_loop,
+    find_exactly_one_terminal_unit,
+)
+from rct229.utils.assertions import getattr_
+from rct229.utils.jsonpath_utils import find_all
+
+EXTERNAL_FLUID_SOURCE = schema_enums["ExternalFluidSourceOptions"]
+FLUID_LOOP_TYPE = schema_enums["FluidLoopOptions"]
```

Comment:
I believe that elsewhere, we have dropped the `_Type` at the end of an enumeration name. So this would be `FLUID_LOOP`.